generator/generate.go

```diff
@@ -17,6 +15,7 @@ import (
 type Config struct {
 	Name          string
 	ExtendMethods []string
+	WorkingDir    string
```

Comment:
> This PR enables working directory support for all packages.Load call. It is required when running unit tests in isolated directories with their own go.mod files - in this case the packages.Load call must be executed inside that directory.

Just to make clear, I understand this correctly. You have a repository with multiple go modules inside, and one goverter call should create converters for all these modules? Was it possible before this change to call goverter multiple times inside the directories of your go module?

Reply:
My use case is slightly different: unit tests. Our codegen tool wraps goverter codegen and invokes it as a library (not via code). When our unit test runs, it creates a temporary directory, drops a go.mod with sources and invokes a codegen (and repeats it few times for different test cases). In each test case we need goverter to work in a scope of that temp (unit test) directory, and ignore the current dir.

P.S. I also noticed by removing go/importer usage the tool became a bit faster. I am not sure what was wrong with the go/importer (other than its lack of support for working dirs), but I did notice a big drop in exec time:
* with go/importer and unit test forced to do chdir before goverter: ~15-20s
* with current code: no need to do chdir: ~3-5s